### PR TITLE
ansible deploy

### PR DESCRIPTION
**Pull request by jfairchild**

### CODE REVIEW

File: ansible/vars/ssh.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+  ricard: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDK9c9kl1dgHgNJqLAC9SxWhMXxx01gtULGYr4SD00HL2yOUdCVLeb/f9bor09LjyI9+SkSGLBjzc7kKACCokB9JuxWSE3aqkRwm+Dlw8JS8wDya6LiB//e2632IJPo38xjMnXiy3gqaWSggjpBf/GuL8GmGQlSHKL5J/igWozeGg5T4T9Zr2U/UzBVIux5SGVnBsehIZjl9Oy3RJ8W5sOulJ0iAGN3w5DUWIt7xNmiyR318wwq/78Rs6nHHUBwkKDpwTTz6+rLFn55GhPNu6/UddZCmx+jLMoCgWvQcV98v7u4do+0S4+kvC7Fas4WbRF+LyC0pp9/9UJ6pdSOajvH'
+  toshiro: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCg4Ye2jYIdSqaDZh8DHmRNO9JQvjEm+RqNxEUXOTFpwXAg0S+jDFwb3QOzyCZoHbsUskrN/lLl8tGbFi1qnIrh/jC56ZBER40GTSrzM78W9Dk4Ed1e2eGMEwilp+W72FgzXMVwbnOTVWZgJNqBBYvjuxOFQfGFxZJdYq9UO+FcUIdL1E5y/PMiapQz21LSANybFojq+4baQVLzW+4ihUiDSEIKS3lIPemArAVchPtsycEYUX/0fCRJ9OTgbnEO4hSSTVwyWkbhLIQfBNmfNfxSio3UBM+/QtqUfK1cs+vNd6fg0H+6MiNQtLsajlA+iUE/sv9rdoMxDq/fNRRTX6tF'
```
Comment (apackin) [inline ansible/vars/ssh.yml]: What happened to my key? 👎

Comment (jfairchild) [reply ansible/vars/ssh.yml]: yes, where did you merge those? mcfly repo?

File: ansible/vars/ssh.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 
 ssh_keys:
+  assaf: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQDQTJwS8RTmFsJh1UYqzwjfmPBqvPRwBZXcHY2V7kgTpIdTij0tnnWLzOVaIyqVMUpXf8UmWwFgMV54VlOmPqmLMG3oN1ZC4rWlBdUoyoBcvVEIkawYHmJIxCpq5FN+OWJl/Nex7lw1X7EvA3rnvzAc9Pd25sSNPmaEn1JsQRJ2tcHUsiB0GmAVw8KOKkd3rHzlwFZUh7fva59dlCl1TeO0AjVK518u0wuO267HXSmcWi3+v4MDGDUhXmlwuZIWImFHsYpFWC/SEQSUzB238NFvedQWuSbfoEbGp8YZCAQfH4qy84QWuvTlUjEhrCCW+A1ImfBaLrIuRE8UIJWCJH'
```
Comment (jfairchild) [inline ansible/vars/ssh.yml]: @apackin found you!

File: ansible/vars/ssh.yml (hunk 1 above)
Comment (mikehow1984) [reply ansible/vars/ssh.yml]: Elmer's a public repo still right? Should the keys be exposed here?

Comment (jfairchild) [reply ansible/vars/ssh.yml]: They are public keys, so it's not a huge problem. This is more exposed than I'd like however.

Can I make elmer private?

Comment (jfairchild) [reply ansible/vars/ssh.yml]: Also, add your key, Mike!

**Pull request merged**